app/change_sets/hyrax/collection_change_set.rb

```diff
@@ -77,10 +75,14 @@ def secondary_terms
       # rubocop:enable Metrics/MethodLength
     end
 
-    # Give the depositor edit access
     def sync
-      self.edit_users = [user.user_key]
-      super
+      # Give the depositor edit access
+      self.edit_users += [user.user_key]
```

Comment:
Should this guard against duplicate entries of `user.user_key` in `edit_users`?

Reply:
I believe Valkyrie::Set is already doing that for us.